en/user_guide/global_page_blocks.rst

```diff
@@ -0,0 +1,33 @@
+Global Pages and Blocks
+=======================
+
+Global Pages and Blocks concept
+-------------------------------
+
+In Open Orchestra, one of the RAD aspects is to simplify creation of pages by using prefunctionnal blocks.
+These blocks must be configured according to their functionnality (see `Block creation`_ and `Block list`_).
+Some blocks could be founded in many pages with the same configuration parameters like header, footer...
+Global pages are containers which allow to stock global configured blocks.
```

Comment:
ok. Transverse pages is the name that we choose for our global page. It could be something else.So with the left menu image, i prefer not to use transverse. Is it ok?